lib/adapters/FileSystem.js

```diff
@@ -122,24 +139,22 @@ class FileSystem extends AbstractAdapter {
 		const results = await Promise.all(promises);
 
 		// Flatten results
-		return Array.prototype.concat.apply([], results);
+		return Array.prototype.concat.apply([], results).filter(($) => $);
```

Comment:
whats the goal of the additional filter step? Cloning?

Reply:
The new check in line 113 can now lead to `null` entries in the result array. These need to be filtered out now.

Reply:
Ack